defaults/main.yml

```diff
@@ -15,6 +15,8 @@ security_autoupdate_enabled: true
 security_autoupdate_blacklist: []
 
 # Autoupdate mail settings used on Debian/Ubuntu only.
+security_autoupdate_reboot: false
```

Comment:
Can you set this to `"false"` since it should be a string in the settings file? That way Python can't accidentally turn it into a `1` or `yes` or something like that.

Reply:
Yeah sure...

Reply:
Done!